src/app/components/feed/FeedItemMediaDialog.js

```diff
@@ -34,7 +34,7 @@ const FeedItemMediaDialog = ({ itemDbid, classes }) => {
         variables={{
           currentTeamSlug,
           feedDbid: parseInt(feedDbid, 10),
-          projectMediaDbid: parseInt(projectMediaDbid, 10),
+          projectMediaDbid: parseInt(itemDbid, 10),
           itemDbid: parseInt(itemDbid, 10),
```

Comment:
Why are there 2 variables with same value?

Reply:
Yeah, let me combine the two. The explanation is:

- A cluster contains multiple items, let's say, A, B and C
- You want to get the cluster which item B belongs to
- From that cluster, you can get the item C

Since clusters are ephemeral, the API doesn't allow clusters to be retrieved by their ID. Previously this component was being used for different IDs, but now they are always the same, for simplicity.